backend/app/api/endpoints/internal/summary.py

```
@@ -0,0 +1,200 @@
+    if api_key.startswith("Bearer "):
+        api_key = api_key[7:]
+
+    # Hash the key for lookup
+    key_hash = hashlib.sha256(api_key.encode()).hexdigest()
+
+    # Find the key in database
+    db_key = (
+        db.query(APIKey)
+        .filter(
+            APIKey.key_hash == key_hash,
+            APIKey.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `APIKey.is_active:` for truth checks

Replace with `APIKey.is_active`

(E712)